corehq/ex-submodules/casexml/apps/case/sharedmodels.py

```diff
@@ -79,8 +79,26 @@ class CommCareCaseAttachment(LooselyEqualDocumentSchema, CaseAttachmentMixin, Un
     def content_type(self):
         return self.server_mime
 
+    @property
+    def is_present(self):
+        """
+        Helper method to see if this is a delete vs. update
+
+        NOTE this is related to but reversed logic from
+        `casexml.apps.case.xml.parser.CaseAttachment.is_delete`.
+        """
+        # Should this only use `attachment_src` since that is the
+        # only field checked by `from_case_index_update`?
```

Comment:
According to the [casexml spec](https://github.com/dimagi/commcare-core/wiki/casexml20) all of `src`, `from` and `name` should be empty for a deletion. I would say that as long as `src` and `from` are empty we should consider it a deletion.

Note that there is a case in the spec (not implemented by us) where `src` could be empty: if `from = 'inline'`